web/modules/custom/collection/templates/collection-leave-button.html.twig

```diff
@@ -0,0 +1,19 @@
+{#
+/**
+ * @file
+ * Template to render the leave collection button.
+ *
+ * Available variables:
+ * - form: the form build array.
+ *
+ * @ingroup themeable
+ */
+#}
+{# Dropdown menu for leave link. #}
+<button id="you-are-member-button" class="button button--blue-light mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent" type="button">
```

Comment:
These are also theming specific classes. This should be the default template that is overridden by our theme to apply those changes.